index.html

```diff
@@ -23,29 +23,227 @@
+          </ul>
+        </div>
+        <button>See Project</button>
+      </div>
+    </section>
+
+    <section class="portfolio-dashboard">
+      <div class="card-header">
+        <div class="course-header">
+          <img src="./Images/Snapshoot Portfolio-projects.png" alt="" class="image">
+        </div>
+      </div>
+      <div class="card-body">
+        <div class="title">Tonic</div>
+        <div class="links">
+          <ul class="link-items">
+            <li class="canopy">CANOPY</li>
+            <span class="list-item">•</span>
+            <li class="list-item">Back end dev</li>
+            <span class="list-item">•</span>
+            <li class="list-item">2015</li>
+          </ul>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        </div>
+        <div class="courses">
+          <ul class="course-list">
+            <li class="courses-list">html</li>
+            <li class="courses-list">css</li>
+            <li class="courses-list">JavaScript</li>
+          </ul>
+        </div>
+        <button>See Project</button>
+      </div>
+    </section>
+    <section class="portfolio-dashboard">
+        <div class="card-header">
+          <div class="header-image">
+            <img src="./Images/Professional-art-printing.png" alt="Dashboard" class="image">
+          </div>
+        </div>
+        <div class="card-body">
+          <div class="title">Multi-Post <br> Stories</div>
+          <div class="links">
+            <ul class="link-items">
+              <li class="canopy">CANOPY</li>
+              <span class="list-item">•</span>
+              <li class="list-item">Back end dev</li>
+              <span class="list-item">•</span>
+              <li class="list-item">2015</li>
+            </ul>
+          </div>
+          <div class="work-description">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+          </div>
+          <div class="courses">
+            <ul class="course-list">
+              <li class="courses-list">html</li>
+              <li class="courses-list">css</li>
+              <li class="courses-list">JavaScript</li>
+            </ul>
+          </div>
+          <button>See Project</button>
+        </div>
+    </section>
+      <section class="about-myself">
+        <div class="card-header">
+          <div class="about-myself-header">
+            <h1>About <br> Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+              work and
+              experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          </div>
+          </div>
+          <div class="card-body">
+            <p class="social-icons-title">Let's Connect</p>

```

Comment:
Please note that this should be a link tag `<a>` which lets you connect o other people. It ideally should open your contact details or social media platforms.